README.md

```diff
@@ -1,41 +1,92 @@
-# ORAS Documentation
 
-[![Netlify Status](https://api.netlify.com/api/v1/badges/db61db6e-a953-4b02-b5fb-7f04f018b9d8/deploy-status)](https://app.netlify.com/sites/oras-project/deploys)
+<img src="https://github.com/oras-project/oras-www/raw/main/docs/assets/images/oras.png" alt="Oras logo" height ="auto" width="200" />
+<br />
 
-![ORAS](https://github.com/oras-project/oras-www/raw/main/docs/assets/images/oras.png)
+<h1>ORAS Documentation</h1>
+ 
+An ORAS is the web site and documentation for the ORAS project.
+<br>
+Link: [oras.land](https://oras.land/)
+<br>
+Documentation : [Link](/docs/index.md).
 
-Source for ORAS website and documentation
 
-Link: [oras.land](https://oras.land/)
 
-**[The documentation can be found here](/docs/index.md).**
+<br>
 
-## Deployment
+<details>
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#about-the-project">About The Project</a>
+    </li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#installation">Installation</a></li>
+      </ul>
+    </li>
+    <li><a href="#developer-notes">Developer notes</a>
+    </li>
+    <li><a href="#how-to-get-involved">How to Get Involved?</a></li>
+    <li><a href="#contributors">Contributors</a></li>
+    <li><a href="#license">Code Of Conduct</a></li>
+  </ol>
+</details>
```

Comment:
In case it's not known, GitHub by default provides a dropdown for all header sections in the top left. E.g., 
![image](https://user-images.githubusercontent.com/814322/219995399-9241056f-7410-4af7-8010-757abc6d5275.png)

Details is a nice touch, but I usually save it for long bits of code that not everyone would be interested in.

Reply:
Love this feature. Thanks for pointing that out! @vsoch